<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cusRegister</class>
 <widget class="QDialog" name="cusRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>顾客注册 </string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/myimages/icon/注册邀请.png</normaloff>:/myimages/icon/注册邀请.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>570</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="birthdayLabel">
      <property name="text">
       <string>生日： </string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;3ds Light&quot;;
font: 20pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>顾客信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;
border-radius: 3px;
border: 1px solid rgb(0, 0, 0);
font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;
border-radius: 3px;
border: 1px solid rgb(0, 0, 0);
font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>263</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>真实姓名： </string>
   </property>
  </widget>
  <widget class="QLabel" name="sexLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>314</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>性别： </string>
   </property>
  </widget>
  <widget class="QLabel" name="phonenumLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>212</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>手机号： </string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>用户名： </string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>161</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>密码： </string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>375</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>身份证号： </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入真实姓名</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sexComboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="phonenumInput">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入11位手机号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameInput">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordInput">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idInput">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入18位身份证号</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usernameInput</tabstop>
  <tabstop>passwordInput</tabstop>
  <tabstop>phonenumInput</tabstop>
  <tabstop>nameInput</tabstop>
  <tabstop>sexComboBox</tabstop>
  <tabstop>idInput</tabstop>
  <tabstop>dateEdit</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
